xcode/commonFastfile

```diff
@@ -213,6 +224,11 @@ def get_keychain_options(options)
   return {:keychain_name => keychain_name, :keychain_password => keychain_password}
 end
 
+def get_screenshots_options()
+  figmaTokenAndFileOptions = load_options_from("screenshots.yaml")
```

Comment:
can I see example of this file?

Reply:
figmaToken: "21309-287b148b-987a-47b4-b82b-3abdb109260b"
figmaFile: "DBQfGtwvrFVfDOp8svbFetln"